templates/legislatives/newsletter/thanks.html.twig

```diff
@@ -0,0 +1,22 @@
+{% extends 'legislatives/newsletter/_layout.html.twig' %}
+
+{% block page_title 'Vous êtes désormais inscrit à la newsletter ! - Legislatives' %}
+
+{% block legislatives_content %}
+    <main class="legislatives_newsletter__content">
+        <section class="legislatives_newsletter__content__form l__wrapper l__row l__row--top l__row--h-stretch-around l__tablet--col space--40-0">
+            <div class="l__wrapper">
+                <h2 class="text--large b__nudge--bottom-larger">
+                    Vous venez de vous inscrire à la newsletter d'un candidats aux élection legislatives. Merci !
```

Comment:
```suggestion
                    Vous venez de vous inscrire à la newsletter d'un candidat aux élections legislatives. Merci !
```

Reply:
I'm waiting for Laïla to give me the text to put on this page. What I put here is just to test the good working of this page